### PR TITLE
Add AcceptPriceDeviationPpm configuration

### PR DESCRIPTION
**Pull request by lukegao209**
The AcceptPriceDeviationPpm is set at 5% and cannot be configured. This presents a certain price slippage risk for RFQ Swaps. Request to add this configuration option. The default value should be 50,000 (5%)

### CODE REVIEW

File: sample-tapd.conf [hunk 1]
```diff
@@ -310,13 +310,17 @@
 ; (latency, etc)
 ; prometheus.perfhistograms=false
 
-[experimental.rfq]
+[experimental]
```
Comment (guggero) [inline sample-tapd.conf:313]: I'm not sure this change is correct. I think the section headers always need to reflect the full prefix to be valid. The config file parsing library we use is quite particular about these things...

File: sample-tapd.conf [hunk 2]
```diff
@@ -310,13 +310,17 @@
...
 
 ; Price oracle gRPC server address (rfqrpc://<hostname>:<port>)
 ; To use the integrated mock, use the following value:
 ; use_mock_price_oracle_service_promise_to_not_use_on_mainnet
 ; experimental.rfq.priceoracleaddress=
 
+; The default price deviation inparts per million that is accepted by the RFQ negotiator.
```
Comment (guggero) [inline sample-tapd.conf:320]: nit: break before 80 characters.

File: tapcfg/server.go [hunk 3]
```diff
@@ -373,6 +373,7 @@ func genServerConfig(cfg *Config, cfgLogger btclog.Logger,
 			ChannelLister:   walletAnchor,
 			AliasManager:    lndRouterClient,
 			// nolint: lll
+			AcceptPriceDeviationPpm:   rfqCfg.AcceptPriceDeviationPpm,
```
Comment (guggero) [inline tapcfg/server.go:376]: If a `//nolint` comment is above a line, it only counts for that line. But both of these are too long. So either needs a comment each, or put the comment before a block, so it counts for the whole block (so for example before `rfq.ManagerCfg{` on line 369).

**Review by guggero — COMMENTED** [review]
Thanks for the change! Makes sense, just a couple of nits.

File: sample-tapd.conf (hunk 1 above)
Comment (lukegao209) [reply sample-tapd.conf:313]: I’ve tested it. The experimental RFQ feature doesn’t work with [experimental.rfq] ; it shows a ‘no group found’ warning

**Review by guggero — APPROVED** [review]
Linter still not happy. And would be nice if you could squash all the commits into a single one with a message that makes sense outside of the context of PR reviews.
Other than that LGTM :tada:

**Pull request closed without merging**